src/storage/mvcc/reader/scanner/mod.rs

```diff
@@ -148,6 +148,16 @@ impl<S: Snapshot> ScannerBuilder<S> {
         self
     }
 
+    /// Set whether to load commit timestamp when scanning.
+    ///
+    /// Default is false.
+    #[inline]
+    #[must_use]
+    pub fn load_commit_ts(mut self, enabled: bool) -> Self {
```

Comment:
```suggestion
    pub fn set_load_commit_ts(mut self, enabled: bool) -> Self {
```
is more consistent with current naming convention.